.github/workflows/docker-publish.yml

```diff
@@ -92,3 +96,4 @@ jobs:
         image: ${{ env.DOCKER_REPOSITORY_OWNER }}/registry-envoy
         context: .
         file: deployments/envoy/Dockerfile
+        build-args: PROTOBUF_VERSION=${{ env.PROTOC_VERSION }}
```

Comment:
This seems fragile since it would only set the version properly when we run the workflow. I wonder - could we get the protoc version within the Dockerfile by reading tools/PROTO_VERSION.sh ?

Reply:
See line # 88. I'm setting from the file.

Reply:
Now that we're overriding it, I wonder if we should remove the default PROTOBUF_VERSION ARG value from the Dockerfile? Or just update it to match?

Reply:
I was wondering why it couldn't be set here: https://github.com/apigee/registry/blob/30c1d99aaefe83e061e6abd299ee6d8aeb3ebb01/deployments/envoy/Dockerfile#L1

Reply:
We could just run ./tools/FETCH-PROTOC.sh in the Dockerfile

Reply:
Yes, I thought of that, but wasn't sure you wanted to get rid of the PROTOBUF_VERSION ARG. Now that I think of it, though, there's probably no reason to keep it. Ok, I'll do as you suggested.

Reply:
Let's get rid of the arg, yes. I don't think it's helping anywhere.

Reply:
Done.